.gitignore

```diff
@@ -84,6 +84,9 @@ celerybeat-schedule
 # SageMath parsed files
 *.sage.py
 
+# bin directory
+bin/
```

Comment:
This will ignore bin/monit script, I didn't ask for this feature, we can still keep in bin area some scripts, what we don't want is to keep executables over there. Please remove it

Reply:
Thanks, I will keep this in mind.